Live-Snap/Live-Snap/Controllers/SelectPhotoViewController.swift

```diff
@@ -180,10 +180,13 @@ class SelectPhotoViewController: UIViewController, UICollectionViewDataSource, U
         guard let cell = tableView.dequeueReusableCell(withIdentifier: "AlbumTableViewCellReuseIdentifier", for: indexPath) as? AlbumTableViewCell else { return UITableViewCell()}
         cell.setText(photoAlbums[indexPath.row].name)
         cell.setAlbumCount(photoAlbums[indexPath.row].assets.count)
+        
+        let initialSelectionIndexPath = IndexPath(row: 0, section: 0)
+        tableView.selectRow(at: initialSelectionIndexPath, animated: true, scrollPosition: .none)
 
```

Comment:
I realized that this is being called each time a cell is created. Will fix on next push